deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,67 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+.container-opdracht-1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.container-opdracht-2 {
+    display: flex;
+    flex-direction: row;
+}
+
+.container-opdracht-2 li {
+    flex:1;
```

Comment:
in de uitwerkingen doen ze het anders, maar die van jouw werkt ook!

Reply:
In de uitwerkingen gaan ze van een vaste 240px uit en die van mij wordt eigenlijk alles evenwijdig geschaald. Ziet er hetzelfde uit in de resultaat. Die van mij is dan soort van responsive :)